CHANGELOG.md

```diff
@@ -27,6 +27,16 @@
 - `dynatraceexporter`: Make `serialization` package `/internal` (#9097)
 - `attributesprocessor`: Remove log names from filters (#9131)
 
+### 🧰 Bug fixes 🧰
+
+- `filestorageextension`: use correct bbolt options for compaction (#9134)
+- `hostmetricsreceiver`: Use cpu times for time delta in cpu.utilization calculation (#8857)
+- `dynatraceexporter`: Remove overly verbose stacktrace from certain logs (#8989)
+- `googlecloudexporter`: fix the `exporter.googlecloud.OTLPDirect` fature-gate, which was not applied when the flag was provided (#9116)
+- `signalfxexporter`: Fix bug to enable timeouts for correlating traces and metrics (#9101)
+- `windowsperfcountersreceiver`: fix exported values being integers instead of doubles (#9138)
+- `tanzuobservabilityexporter`: Make metrics stanza in config be optional (#9098)
```

Comment:
Something is wrong after the rebase. Also, the fix description should go to around line 16 or so

Reply:
I rebased and pulled the tanzuobservability exporter change up into the unreleased section.